index.html

```diff
@@ -19,7 +19,7 @@
 <body>
     <nav>
         <div class="container0">
-            <a href="#" class='logo'>My Logo</a>
+            <a href="#" class='logo'>AD</a>
```

Comment:
- Please wrap your `nav` tag in a `header` tag, this will help improve the accessibility of your site. 😄 😄